src/DependencyInjection/FOSElasticaExtension.php

```diff
@@ -93,14 +92,16 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         $this->loadClients($config['clients'], $container);
-        $container->setAlias('fos_elastica.client', \sprintf('fos_elastica.client.%s', $config['default_client']));
-        $container->getAlias('fos_elastica.client')->setPublic(true);
-        $container->setAlias(ElasticaClient::class, new Alias('fos_elastica.client', false));
-        $container->setAlias(Client::class, 'fos_elastica.client');
-        $container->getAlias(Client::class)->setPublic(false);
+        $container->setAlias('fos_elastica.client', \sprintf('fos_elastica.client.%s', $config['default_client']))
+            ->setPublic(false);
+        $container->setAlias(ElasticaClient::class, 'fos_elastica.client')
```

Comment:
You don't choose `$container->setAlias(ElasticaClient::class, new Alias('fos_elastica.client', false))` finally ?

Reply:
Well, the other aliases https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/src/DependencyInjection/FOSElasticaExtension.php#L99-#L100 do not use the Alias class either, so it was more easier and more consistent to remove this one.

Reply:
You're right, can be improved in another PR to make this consitstent.